### ISSUE TITLE
Clean up unused controllers/services

### ISSUE THREAD

**Issue by Mathyn** [Smilo-platform/Smilo-Api#41]
Looking at the API it appears the BalanceController is no longer used (superseded by the AddressController).

The PriceController is a dummy right now and the StoredCoinService is not used (and will look very different in its final form). Since it might be a long time before we start using these wouldn't it be better to just remove them right now? Makes for a cleaner code base.
